app/controllers/feedback_controller.rb

```diff
@@ -15,6 +16,11 @@ def update
     end
   end
 
+  # @return [User | Guest]
+  def respondant
```

Comment:
Is `current_user` sometimes `nil`, if so this pattern is normally used to ensure it never is. Does doing that work for us @jack-coggin ?

Reply:
I have made this change now